### PR TITLE
Fullscreen mode on modelling-submission.component

### PR DESCRIPTION
**Pull request by TobiasPr**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.
- [x] Client: I documented the TypeScript code using JSDoc style.
- [x] Client: I added multiple screenshots/screencasts of my UI changes
- [x] Client: I translated all the newly inserted strings into German and English

### Motivation and Context

This change implements the feature request for #470 
It provides the possibility to toggle full screen mode for modelling exercises, so that students can present their modelling solutions. It also includes the feedback of the tutor on the side

### Description

added a fullscreen module in shared code which can provide this functionality to all wrapped children. I also included it in the modelling-submission.component

### Steps for Testing

[Create a modelling exercise]
1. Log in to Artemis
2. Navigate to Course Management
3. Select "Exercises" of your test course
4. Create a new Modelling Exercise

[Create a manual graded submission]
1. Go to Course Overview
2. Select your test course
3. Start Modelling Exercise
4. Do Modelling Exercise
5. Submit Modelling Exercise
6. Go to Course Management 
7. Select Tutor-Dashboard of your Exercise
8. Participate in Assessment
9. Assess the previous submitted modelling exercise

[Navigate to fullscreen]
1. Navigate to Course Overview
2. Select your test course
3. Go to you modelling exercise
4. Click View Results
5. On the bottom right of the assessment section, you will find the toggle Fullscreen button
6. See if fullscreen mode works

### Screenshots

fullscreen_modelling_editor

### CODE REVIEW

**Comment by krusche** [comment]
Looks good, I like it 👍  

3 questions:

1) Does it also work, if there is no assessment yet? (it would be good if it would always be possible to go into full screen on this page, also e.g. while users are modeling or if there is no assessment yet)
2) I think the placement of the fullscreen button is a bit unintuitive in the right bottom and might be hard to find. Can we place it in the upper right corner (above the assessment)?
3) Can we have two versions of the button (e.g. compact and extended)? Compact would be the small version of the button that you have now and extended would be a normal button with the text "Fullscreen" or "Presentation mode" which would be more intuitive in this scenario. Then developers could choose which version to use when embedding the button.

**Comment by TobiasPr** [comment]
Looks like this now:

Assessment mode - tutors (no button):
screenshot_3

Student drawing:
screenshot_5

Student reviewing assessment:
screenshot_4

Full screen of extended button:
image

**Comment by krusche** [comment]
Looks great. Thanks!

One small suggestion, in Fullscreen Mode, it would be good if the button changes to "Exit Fullscreen" and if you click again on it, you can also exit the fullscreen.

**Comment by krusche** [comment]
@TobiasPr unfortunately, we have merge conflicts. Could you please resolve them?
Maybe @HanyaElhashemy can help here?

**Comment by TobiasPr** [comment]
> @TobiasPr unfortunately, we have merge conflicts. Could you please resolve them?
> Maybe @HanyaElhashemy can help here?

@krusche Done - one thing i noticed though, after merging develop and doing yarn install there is one change in yarn.lock which is not committed, should I do that here or is it more clean in a separate branch?

File: src/main/webapp/i18n/de/modelingEditor.json [hunk 1]
```diff
@@ -9,6 +9,11 @@
             "submitSuccessfulWithAssessment": "Dein Diagramm wurde erfolgreich abgesendet! Du kannst dein Diagramm nun nicht mehr bearbeiten. Dein Feedback wird unten angezeigt.",
             "submitDeadlineMissed": "Dein Diagramm wurde erfolgreich abgesendet! Du kannst es ändern oder auf dein Feedback warten. Da du die Deadline verpasst hast wird deine Bewertung nicht gezählt.",
             "empty": "Ein leeres Diagramm kann nicht abgegeben werden.",
+            "fullscreen": {
+                "enterButtonText": "Fullscreen",
```
Comment (krusche) [inline src/main/webapp/i18n/de/modelingEditor.json:13]:
```suggestion
                "enterButtonText": "Vollbild",
```

File: src/main/webapp/i18n/de/modelingEditor.json [hunk 2]
```diff
@@ -9,6 +9,11 @@
...
+                "exitButtonText": "Fullscreen verlassen",
```
Comment (krusche) [inline src/main/webapp/i18n/de/modelingEditor.json:14]:
```suggestion
                "exitButtonText": "Vollbild verlassen",
```

File: src/main/webapp/i18n/de/modelingEditor.json [hunk 3]
```diff
@@ -9,6 +9,11 @@
...
+                "tooltip": "Fullscreen"
```
Comment (krusche) [inline src/main/webapp/i18n/de/modelingEditor.json:15]:
```suggestion
                "tooltip": "Vollbildmodus"
```

**Pull request merged**